raiden-cli/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint --max-warnings 0 --ext '.ts' .",
     "build": "tsc",
-    "build:bundle": "webpack --progress",
+    "build:bundle": "TS_NODE_PROJECT=./ignored ncc build -C ./src/index.ts -o ./bundle",
```

Comment:
webpack was not detecting `leveldown` native dependency to include in the bundle (required since database refactor), required a complex configuration and was not playing well with `yargs` dependency. `ncc` uses webpack under the hood, but worked out of the box without config.